README.md

```diff
@@ -54,6 +54,25 @@ $middleware->add('Muffin/Footprint.Footprint');
 It must be added **after** `AuthenticationMiddleware` to ensure that it can read
 the identify info after authentication is done.
 
+If you don't have direct access to the place where `AuthenticationMiddleware` is enqueued you can instead enqueue it via adding this to your `src/Application.php`
+
+```php
+use Authentication\Middleware\AuthenticationMiddleware;
+use Cake\Event\EventInterface;
+use Cake\Event\EventManager;
+use Cake\Http\MiddlewareQueue;
+use Muffin\Footprint\Middleware\FootprintMiddleware;
+
+// inside the bootstrap() method
+$eventManager = EventManager::instance();
```

Comment:
You can just use `$this->getEventManager()`. Also I don't want to clutter the default setup by adding this edge case info here. Just put a single statetement "If you don't ... then check [here]" and a section just above the "Patches & Features".